lib/line-protocol.js

```diff
@@ -169,13 +170,8 @@ module.exports.format = (event, config) => {
         return;
     }
 
-    const tags = internals.serialize(internals.tags(event));
+    const tags = internals.serialize(internals.tags(event, config));
     const eventValues = getEventValues(event);
-    if (config.metadata) {
-        Object.keys(config.metadata).forEach( (key) => {
-            eventValues[key] = internals.String(config.metadata[key]);
-        });
-    }
```

Comment:
These changes break some of the project's tests.  When I ran them locally I got this failure.

```
$ npm run test
  ...
Error: Expected 'ops,host=mytesthost,pid=9876,serviceName=undefined os.cpu1m=1.8408203125,os.cpu5m=1.44287109375,os.cpu15m=1.15234375,os.freemem=162570240i,os.totalmem=6089818112i,os.uptime=11546i,proc.delay=0.07090700045228004,proc.heapTotal=41546080i,proc.heapUsed=27708712i,proc.rss=55812096i,proc.uptime=18.192 123456789000000' to equal specified value
    at internals.equal (/Users/alecferguson/proj/good-influx/node_modules/code/lib/index.js:266:17)
    at dataRows.forEach (/Users/alecferguson/proj/good-influx/test/index.test.js:76:38)
    at Array.forEach (native)
    at IncomingMessage.req.on (/Users/alecferguson/proj/good-influx/test/index.test.js:75:26)
    ...
```

You probably need to change `const expectedMessage` in `index.test.js`.

Granted, this project does not have CI set up, so we're stuck running them locally.  Please make sure they pass for you.